packages/shared/src/hooks/feed/useFeedContextMenu.ts

```diff
@@ -62,21 +65,21 @@ export default function useFeedContextMenu(): FeedContextMenu {
     column: number,
   ) => {
     if (postMenuIndex === index) {
-      setPostMenuLocation(null);
+      setPostShareMenuLocation(null);
       return;
     }
-    setPostMenuLocation({ index, row, column });
-    const { right, bottom } = e.currentTarget.getBoundingClientRect();
-    showShareMenu(e, {
-      position: { x: right, y: bottom + 4 },
```

Comment:
Here as well, I think we'll need this right?

Reply:
same as above, this is already handled by `onMenuClick`: https://github.com/dailydotdev/apps/blob/b3d1c8d32112083cdc6fd8c0a3220f8dd4a8e2fe/packages/shared/src/hooks/useContextMenu.ts#L41-L53